### DISCUSSION TITLE
Linking parent and child IDs

### DISCUSSION THREAD

**Discussion by juliakatryan** [now-i-know-my-abcd/docs#54] [Where in the World is…? (Help)] [answered]
How do I know which parents belong to which child? I can't seem to find this information anywhere. I've searched here, I've searched for information on the NDAR GUID, I've searched research papers supplementary materials, I've searched the Now I Know my ABCs blog, the Github for data cleaning code, and the ABCD.org webiste. Any other leads/pointers would be greatly appreciated.

**Comment by mangstad** [comment, marked as answer]
Can I ask what you mean by which parents go with which children? At least for all the data files I've worked with, rows are always tagged with a src_subject_id, which corresponds to the child's ID. Even in questionnaires that are about the parents (like the adult behavior checklist), this src_subject_id field still refers to the child's ID.

**Comment by juliakatryan** [comment]
Got it, I think that answers my question. I wasn’t aware that the child ID
also corresponds to the parents ID, that makes a lot of sense! Thank you!
